### DISCUSSION TITLE
where can I get the  Operation document   of  PostgreSQL?

### DISCUSSION THREAD

**Discussion by longyunqi** [dbeaver/dbeaver#15658] [Q&A]
I want to use dbeaver,but I  can not find documention to tell me how to use in dbeaver with PostgreSQL.

**Comment by LonwoLonwo** [comment]
Hello @longyunqi 

You can find our documentation here: https://github.com/dbeaver/dbeaver/wiki
Or here: https://dbeaver.com/docs/wiki/
And PostgreSQL documentation you can find here: https://www.postgresql.org/docs/

Here you can find the info about new connection creating: https://github.com/dbeaver/dbeaver/wiki/Create-Connection
About work with Database Navigator Tree - https://github.com/dbeaver/dbeaver/wiki/Database-Navigator
Or about data viewing and editing - https://github.com/dbeaver/dbeaver/wiki/Data-Viewing-and-Editing

If you have other specific questions - you can ask here. And we will try to answer.
